### ISSUE TITLE
Shortcuts displayed for formatting options are Mac-specific

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/platform-forms-client#3035]
# Issue / opportunity

The bold and italics options in the WYSIWYG editor shows `Bold (⌘B)` and `Italic (⌘I)` which are apple/mac/ios shortcuts, even when accessed on pc/windows or other devices.

Consider removing the parentheses with the shortcut, which could cause more confusion for users who are probably more often using pc/windows/edge/tablet?

# Screenshots

![Screenshot 2023-12-15 at 2 33 56 PM](https://github.com/cds-snc/platform-forms-client/assets/38330843/1a684158-b453-4663-a79d-1982cc2ed138)

![Screenshot 2023-12-15 at 2 34 01 PM](https://github.com/cds-snc/platform-forms-client/assets/38330843/d59704b8-883f-4d8f-ab63-3042c64677af)

**Comment by connorscarolyns** [comment]
Was this tested on a PC? I think the key commands are essential for AT users.

**Comment by anikbrazeau** [comment]
Yes, this issue was observed in a Fable session by Notify and also confirmed for Forms on the tablet afterward.

On PC, the symbol is still the command key `⌘` symbol. 

Is there a way to differentiate between the two and display `Ctrl` to match expectations users based on their device?

**Comment by samsadasivan** [comment]
As a short-term fix: lets do both (⌘+B or Ctrl+B), If we can track device type, then we can cater the personalization but I don't want to remove it completely. cc: @timarney

**Comment by timarney** [comment]
We can do a quick experiment using the userAgent string (via the browser) --- if that doesn't work or will take some time yes to https://github.com/cds-snc/platform-forms-client/issues/3035#issuecomment-2794386030

**Comment by Abi-Nada** [comment]
This issue has been rolled in with the lexical update #5265
